### ISSUE TITLE
Passing str to functions annotated AnyStr shows squiggly in .pyi files

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/pylance-release#243]
## Environment data

-   Language Server version: 2020.8.1
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.3

## Expected behaviour

Should not show squiggles

## Actual behaviour

This occured in a auto-generated pyi file in strict mode. There might be two problems here:
1. It is generating actual code in .pyi files
2. the squiggles

![image](https://user-images.githubusercontent.com/3840081/90296549-a7a36d00-de40-11ea-82dc-de3d0f7caab2.png)

![image](https://user-images.githubusercontent.com/3840081/90296604-cbff4980-de40-11ea-8039-38d39359146e.png)

Contents of the actual generated file below in the code snippet section.

## Logs

```
XXX
```

## Code Snippet / Additional information

Put this in a pyi file. Turn on strict mode.
```python
"""
This type stub file was generated by pyright.
"""

import os

haarcascades = os.path.join(os.path.dirname(__file__), "")

```

**Comment by erictraut** [comment]
Thanks for the bug report. I've eliminated the incorrect error.

That said, stub files should not include statements like this. They should include only type information, not logic that's manipulating file names or paths.

**Comment by jakebailey** [comment]
Yep, this code just came out of the stub generator (which is another issue in and of itself).

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202082-20-august-2020
